Ohshie.MathGame/OhshieMathGame/GameModes/CareerGameMode.cs

```diff
@@ -0,0 +1,168 @@
+            
+        }
+        GameFinished();
+    }
+    private static void GameFinished()
+    {
+        Console.Clear();
+        if (Lives < 1)
+        {
+            Console.WriteLine("Dang! You run out of lives.\n" +
+                              $"You've solved {GameController.Score} equations our of possible 25");
+            GameController.ScorePrinter();
+            
+            Console.WriteLine("Press enter to go back to menu");
+            
+            Console.ReadLine();
+        }
+        else if (Lives > 0 && GameController.GamesPlayed == 26)
+        {
+            Console.WriteLine("You WON\n" +
+                              $"You've solved {GameController.Score} equations our of possible 25\n" +
+                              $"You still have {Lives} lives left");
+            GameController.ScorePrinter();
+            
+            Console.WriteLine("Press enter to go back to menu");
+            
+            Console.ReadLine();
+        }
+    }
+    private static void ProblemGenerator(int amountOfVariables)
+    {
+        // getting maximum amount of variables possible from settings
+        
+        double[] variables = new double[amountOfVariables];
+        
+        // Creating random numbers for equation
+        
+        for (int i = 0; i < variables.Length; i++)
+        {
+            variables[i] = _random.Next(1, 10);
+        }
+        
+        // getting random operators for that equation
+        
+        string[] operatorsInEquation = new string[variables.Length - 1];
+        
+        for (int i = 0; i < operatorsInEquation.Length; i++)
+        {
+            operatorsInEquation[i] = GameController.OperatorsInPlay[_random.Next(0,GameController.OperatorsInPlay.Count)];
+        }
+        
+        // filling a sting
+        
+        GameController.Equation = "";
+        
+        for (int i = 0; i < variables.Length; i++)
+        {
+            GameController.Equation += variables[i];
+            if (i<operatorsInEquation.Length)
+            {
+                GameController.Equation += operatorsInEquation[i];
+            }
+        }
+
+    }
+    private static float PlayerSolution()
+    {
+        float playerAnswer;
+        // while loop for safety check
+        while (true)
+        {
+            Console.Clear();
+            Console.WriteLine($"Correct answers {GameController.Score}. {Lives} lives left\n" +
+                              $"Problem: {GameController.GamesPlayed}. Difficulty level: {_difficultyLevel}");
+            Console.WriteLine("Solve this:");
+            Console.Write(GameController.Equation+"=");
+                
+            if (Single.TryParse(Console.ReadLine(),out playerAnswer))
+                break;
+            Console.WriteLine("Looks like you entered something that is not a number.\n" +
+                              "Try again!");
+        }
+        return playerAnswer;
+    }
+    
```

Comment:
Please remove empty lines from 162 from 167. [Check out our code conventions on spacing](https://thecsharpacademy.com/article/58) and adjust the rest of the project: 

![image](https://user-images.githubusercontent.com/108711476/220533509-c2d46a70-21fb-4e4f-ba31-586aa5067292.png)